shared/actions/config/index.js

```diff
@@ -68,7 +69,7 @@ export function bootstrap (): AsyncAction {
       })
     } else {
       Promise.all(
-        [dispatch(getCurrentStatus()), dispatch(getExtendedStatus()), dispatch(getConfig())]).then(() => {
+        [dispatch(getCurrentStatus()), dispatch(getExtendedStatus()), dispatch(getConfig()), dispatch(favoriteList())]).then(() => {
```

Comment:
I lack context on the architectural thoughts around this, but seems like a fine place to put it.